integration-tests/modules/__tests__/stock-location/index.spec.ts

```diff
@@ -52,5 +52,33 @@ medusaIntegrationTestRunner({
         )
       })
     })
+
+    describe("Update stock locations", () => {
+      let stockLocationId
+
+      beforeEach(async () => {
+        const createResponse = await api.post(
+          `/admin/stock-locations`,
+          {
+            name: "test location",
+          },
+          adminHeaders
+        )
+
+        stockLocationId = createResponse.data.stock_location.id
+      })
+      it("should update stock location name", async () => {
+        const response = await api.post(
+          `/admin/stock-locations/${stockLocationId}`,
```

Comment:
todo: I realize we have 3-4 PRs containing API tests in `modules`, so I think now would be a good time to migrate these to `integration-tests/api` to not accumulate too much debt.